packages/conf-pic-switch/conf-pic-switch.0.1/opam

```diff
@@ -1,3 +1,12 @@
-opam-version: "1"
+opam-version: "1.2"
+authors: ["Francois Berenger"]
 maintainer: "francois.berenger@inria.fr"
-build: [ ["./check.sh"] ]
+homepage: "https://github.com/ocaml/opam-repository"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+dev-repo: "https://github.com/ocaml/opam-repository.git"
+build: [
+  ["./check.sh"]
+]
+available: [
+  ocaml-version >= "4.01.0" & < "4.02.2" & !preinstalled
```

Comment:
I think you need to add parenthesis:
```
available: [
   (ocaml-version >= 4.01.0)
& (ocaml-version < 4.02.2)
& !preinstalled
]
```

(I'm not 100% sure with the syntax in recent versions of opam, need to check the manual. /cc @AltGr)